app/models/rubygem.rb

```diff
@@ -262,10 +262,6 @@ def find_or_initialize_version_from_spec(spec)
     version
   end
 
-  def first_created_date
-    versions.by_earliest_created_at.first.created_at
```

Comment:
can we please remove this version method as well? seems to be the only place where it was used.